public-ui/src/app/data-browser/components/survey-version-table/survey-version-table.component.css

```diff
@@ -30,7 +30,6 @@ strong {
     font-family: GothamBold;
 }
 .version-box-body{
-    height: 8rem;
```

Comment:
![image](https://user-images.githubusercontent.com/21062724/96480136-79a32600-11ff-11eb-9328-5da8d73cb22b.png)

This was creating extra space in the table.